sched/scheduler/stateful_scheduler.go

```diff
@@ -55,7 +55,7 @@ const DefaultMaxJobsPerRequestor = 100
 const DefaultSoftMaxSchedulableTasks = 1000
 
 // Increase the NodeScaleFactor by a percentage defined by 1 + (Priority * NodeScaleAdjustment)
-var NodeScaleAdjustment = .5
+var NodeScaleAdjustment = 1.0
```

Comment:
i still don't grok why this gets changed around

Reply:
We've been running with P1 for long enough to be confident about it imo (at least a week I believe). This adjustment is elaborated on a bit more in GetNodeScaleFactor().

In practice this change means that a P1 job will get at least 2x the nodes of a P0 jobs, rather than 1.5x.